profitbricks-rest/src/main/java/org/apache/jclouds/profitbricks/rest/domain/Snapshot.java

```diff
@@ -0,0 +1,251 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.jclouds.profitbricks.rest.domain;
+
+import java.util.Date;
+
+import com.google.auto.value.AutoValue;
+
+import org.jclouds.javax.annotation.Nullable;
+
+@AutoValue
+public abstract class Snapshot {
```

Comment:
In relation to this [comment](https://github.com/jclouds/jclouds-labs/pull/227#issuecomment-172589850), it seems you can remove this Domain class, at least for this PR. I'd understand the inclusion of Volume and Nic, since they're returned by the API, based on [docs](https://devops.profitbricks.com/api/rest/#retrieve-a-server).